File: client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/TestResultsDto.java
package org.evomaster.client.java.controller.api.dto;

import org.evomaster.client.java.instrumentation.shared.dto.ControlDependenceGraphDto;

import java.util.ArrayList;
import java.util.List;

public class TestResultsDto {

    public List<TargetInfoDto> targets = new ArrayList<>();


    /**
     * This list is sorted based on the action indices
     */
    public List<AdditionalInfoDto> additionalInfoList = new ArrayList<>();

    public List<ExtraHeuristicsDto> extraHeuristics = new ArrayList<>();

    /**
     * Incremental DynaMOSA control-dependence graphs discovered since the last handshake.
     */
    public List<ControlDependenceGraphDto> dynamosaCdgs = new ArrayList<>();
}

File: outdated/outdated-client-js/evomaster-client-js/src/controller/api/dto/TestResultsDto.ts
import AdditionalInfoDto from "./AdditionalInfoDto";
import ExtraHeuristicsDto from "./ExtraHeuristicsDto";
import TargetInfoDto from "./TargetInfoDto";

export default class TestResultsDto {

    public targets: TargetInfoDto[] = [];

    public additionalInfoList = new Array<AdditionalInfoDto>();

    public extraHeuristics = new Array<ExtraHeuristicsDto>();
}
